src/access/profiling/access_configs.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2025 ACCESS-NRI and contributors. See the top-level COPYRIGHT file for details.
+# SPDX-License-Identifier: Apache-2.0
+
+import logging
+from pathlib import Path
+
+from access.profiling.cice5_parser import CICE5ProfilingParser
+from access.profiling.config_profiling import ProfilingLog
+from access.profiling.fms_parser import FMSProfilingParser
+from access.profiling.payu_config import PayuConfigProfiling
+from access.profiling.um_parser import UMProfilingParser
+
+logger = logging.getLogger(__name__)
+
+
+class ESM16ConfigProfiling(PayuConfigProfiling):
+    """Handles profiling of ACCESS-ESM1.6 configurations."""
+
+    def get_component_logs(self, path: Path) -> dict[str, ProfilingLog]:
+        """Returns available profiling logs for the components in ACCESS-ESM1.6.
+
+        Args:
+            path (Path): Path to the output directory.
+        Returns:
+            dict[str, ProfilingLog]: Dictionary mapping component names to their ProfilingLog instances.
+        """
+        logs = {}
+        um_logfile = path / "atmosphere" / "atm.fort6.pe0"
```

Comment:
This filename is specified in `atmosphere/um_env.yaml` as `UM_STDOUT_FILE=atm.fort6.pe` [here](https://github.com/ACCESS-NRI/access-esm1.6-configs/blob/dev-preindustrial%2Bconcentrations/atmosphere/um_env.yaml#L17) and then rank=0 appends the 0 (presumably)